docs/eventing/samples/writing-event-source-easy-way/README.md

```diff
@@ -0,0 +1,298 @@
+# Introduction
+
+As stated in [tutorial on writing a Source with a Receive Adapter](../writing-receive-adapter-source/README.md), there are multiple ways to
+create event sources. The way in that tutorial is to create an independent event source that has its own CRD.
+
+In this tutorial though, you will build an event source in Javascript and use it with
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) and / or [SinkBinding](../../../eventing/sources/README.md#meta-sources).
+
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) is an easy way to turn any dispatcher container into an Event Source.
+Similarly, another option is using [SinkBinding](../../../eventing/sources/README.md#meta-sources)
+which provides a framework for injecting environment variables into any Kubernetes resource which has a `spec.template` that looks like a Pod (aka PodSpecable).
+
+Code for this tutorial is available [here](./).
+
+# Bootstrapping
+
+Create the project and add the dependencies:
+```bash
+npm init
+npm install cloudevents-sdk --save
+```
+
+## Making use of ContainerSource
+
+`ContainerSource` and `SinkBinding` both work by injecting environment variables to an application.
+Injected environment variables at minimum contain the URL of a sink that will receive events.
+
+Following example emits an event to the sink every 1000 milliseconds.
+The sink URL to post the events will be made available to the application via the `K_SINK` environment variable by `ContainerSource`.
+
+```javascript
+// File - index.js
```

Comment:
You may want to use the [`{{% readfile file="index.js" %}}`](https://github.com/knative/website/blob/master/layouts/shortcodes/readfile.md) shortcode here.

Reply:
Let's do that later if necessary. The code I embedded in the tutorial is a bit shorter and more focused than the code in the files.